{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -6,25 +6,15 @@ skipsdist = True
 
 [testenv]
 setenv =
-    DJANGO_SETTINGS_MODULE={{ cookiecutter.project_slug }}.settings.base
+    DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.config.settings.test
     PYTHONPATH={toxinidir}
-    SECRET_KEY=test
     DATABASE_URL=postgresql://{{ cookiecutter.project_slug }}:{{ cookiecutter.project_slug }}@localhost/{{ cookiecutter.project_slug }}
-deps = -r{toxinidir}/requirements/base.txt
+deps = -r{toxinidir}/requirements/dev.txt
 whitelist_externals = scripts/check_migrations.sh
 commands=
     scripts/check_migrations.sh
+    pytest --cov-report=term --cov={{ cookiecutter.project_slug }} --cov=apps -n 2
```

Comment:
There's no more `apps` directory with the new structure. All the "apps" will be directly in the `{{ cookiecutter.project_slug }}` directory.